oat-sa/tao-core
# Feature/tao 7265 datetime picker component

This Pull Request is the first in the attempt to replace the current date/time picker based on jqueryui by a new one (without the dependency to jqueryui).

This PR contains the following : 
 - Adding the flatpickr library into tao-core
 - Create a dedicated component that let's you use flatpickr
 - Update style for the picker to match TAO guidelines, including control buttons
 - The unit test that covers the component features (plus a visual playground)
 - The replacement of the picker in the ontologies based forms

How to test it : 
 1. Run the test in a browser :  `${your tao localdomain}/tao/views/js/test/ui/datetime/picker/test.html`
 2. Run the test using the CLI : `cd tao/views/build && npx grunt connect:test qunit:single --test=/tao/views/js/test/ui/datetime/picker/test.html`
 3. Test it in the ontology form : 
First create a proprety on a class with the "Calendar widget" 
![image](https://user-images.githubusercontent.com/468620/55544494-861e6700-56cb-11e9-8691-0b63a95e3b1a.png)
Create instances and instead of this picker : 
![image](https://user-images.githubusercontent.com/468620/55544547-9d5d5480-56cb-11e9-8d3b-9f21582f2d77.png)
You should see this picker : 
![image](https://user-images.githubusercontent.com/468620/55544362-36d83680-56cb-11e9-8ce5-ae1b612d2d29.png)
